public/tracking.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<!-- Piwik -->
+		<script type="text/javascript">
+			var _paq = _paq || [];
+			_paq.push(['setDocumentTitle', document.domain + "/" + document.title]);
+			_paq.push(['trackPageView']);
+			_paq.push(['enableLinkTracking']);
+			(function() {
+				var u="https://stats.data.gouv.fr/";
+				_paq.push(['setTrackerUrl', u+'piwik.php']);
+				_paq.push(['setSiteId', 4]);
+				var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+				g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
+			})();
+		</script>
+		<!-- End Piwik Code -->
+		<link rel="icon" href="favicon.ico">
+
+		<title>OpenFisca - Turn law into software</title>
+
+		<link rel="stylesheet" type="text/css" href="css/main.css">
+	</head>
+	<body>
+		<img id="logo" src="logo-openfisca.svg" alt="OpenFisca">
+
+
+		<h1>Cookies and opt-out</h1>
+		<p>
+			When you visit this website, we leave a small text file (a "cookie") on your computer. This helps us measure how many visits we've had and what pages are most viewed.
+		</p>
+		<div id='optout'>
```

Comment:
Double quotes in HTML. Please add a linter at least in your local dev environment to prevent these from happening again.